### PR TITLE
Step registry ui: Add handler for ci-operator config reference

### PR DESCRIPTION
**Pull request by alvaroaleman**
The content for the handler is pre-generated because the generation needs the source code which we do not have anymore in the final image.

### CODE REVIEW

File: pkg/webreg/zz_generated.ci_operator_reference.go [hunk 1]
```diff
@@ -0,0 +1,1210 @@
...
+	"                namespace: ' '\n" +
+	"                tag: ' '\n" +
+	"            optional_on_success: false\n" +
+	"            # Reference is the name of a step reference.\n" +
+	"            ref: \"\"\n" +
+	"            # Resources defines the resource requirements for the step.\n" +
+	"            resources:\n" +
+	"                # LiteralTestStep is a full test step definition.\n" +
+	"                limits:\n" +
+	"                    # LiteralTestStep is a full test step definition.\n" +
```
Comment (stevekuznetsov) [inline pkg/webreg/zz_generated.ci_operator_reference.go:1196]: This don't look quite right

Comment (alvaroaleman) [reply pkg/webreg/zz_generated.ci_operator_reference.go:1196]: How so?

Comment (stevekuznetsov) [reply pkg/webreg/zz_generated.ci_operator_reference.go:1196]: This string "LiteralTestStep..." is repeated like 400 times on unrelated fields in here

**Comment by stevekuznetsov** [comment]
mechanisms for generating and serving this LGTM but the actual content has bugs

/lgtm

File: pkg/webreg/zz_generated.ci_operator_reference.go (hunk 1 above)
Comment (alvaroaleman) [reply pkg/webreg/zz_generated.ci_operator_reference.go:1196]: Hm, that doc comes from a promoted field: https://github.com/openshift/ci-tools/blob/581ca445940d60c43bce33451236e3ce8d134cab/pkg/api/types.go#L650

Seems like a bug in genyaml. Do you mind merging this already withotu advertising it and I will look into the genyaml stuff later?

**Pull request merged**